tests/osaka/eip7692_eof_v1/eip7873_tx_create/test_txcreate_failures.py

```diff
@@ -619,21 +569,16 @@ def test_static_flag_txcreate(
     initcode_hash = keccak256(initcode)
     sender = pre.fund_eoa()
     contract_address = pre.deploy_contract(
-        code=Container(
-            sections=[
-                Section.Code(
-                    code=Op.TXCREATE(tx_initcode_hash=initcode_hash, value=endowment) + Op.STOP,
-                ),
-            ]
-        )
+        code=Op.TXCREATE(tx_initcode_hash=initcode_hash, value=endowment) + Op.STOP,
     )
     calling_code = (
         Op.SSTORE(slot_call_result, opcode(address=contract_address))
         + Op.SSTORE(slot_code_worked, value_code_worked)
         + Op.STOP
     )
     calling_address = pre.deploy_contract(
-        Container.Code(calling_code) if opcode == Op.EXTSTATICCALL else calling_code
+        Container.Code(calling_code) if opcode == Op.EXTSTATICCALL else calling_code,
+        evm_code_type=EVMCodeType.EOF_V1 if opcode == Op.EXTSTATICCALL else EVMCodeType.LEGACY,
```

Comment:
hm this worked also without explicit `evm_code_type` argument?

Reply:
unfortunately not - if you leave the 1st arg as legacy it will force-wrap it to EOF, even despite we only really "want" it on the TXCREATE code.

However, what is redundant now is maybe the `if` just above, missed that

Reply:
yup, redundant, removed, left a comment to clarify :)